src/logger.ts

```diff
@@ -1,7 +1,17 @@
 import pino from "pino";
 
+const urlToRedacted = (url: string) => {
+  if (url.includes("/api/v1/account-history")) {
+    return "/api/v1/account-history";
+  }
+  if (url.includes("/api/v1/account-balances")) {
+    return "/api/v1/account-balances";
+  }
+  return url;
```

Comment:
looks like `/token-details` also takes a `pub_key` query param. Should we strip that, too?

Reply:
Thanks for catching! Added in 34ac067a494927be8cfc15ea9cadfbfb2c851883